EFCore.NamingConventions/NamingConventions/Internal/NameRewriterBase.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore.Metadata.Conventions;
+
+namespace EFCore.NamingConventions.Internal
+{
+    /// <summary>
+    /// This class only required so we can have common superclass for all name rewriters
+    /// </summary>
+    internal abstract class NameRewriterBase : IEntityTypeAddedConvention, IPropertyAddedConvention
```

Comment:
This is not a bad idea, for forcing new naming conventions to implement all relevant methods.